### PR TITLE
Deprecate using master and slave

### PR DESCRIPTION
**Pull request by jordisala1991**
## Subject

I am targeting this branch, because this is BC.

## Changelog

```markdown
### Deprecated
- Deprecate using `master` and `slave` for filesystem and cdn configuration, use `primary` and `secondary` instead.
```

### CODE REVIEW

File: tests/Filesystem/ReplicateTest.php [hunk 1]
```diff
@@ -21,40 +21,40 @@ class ReplicateTest extends TestCase
 {
     public function testReplicate(): void
```
Comment (ChristianGripp) [inline tests/Filesystem/ReplicateTest.php:22]: Can you add a test that checks the BC layer

**Review by core23 — CHANGES_REQUESTED** [review]

File: tests/Filesystem/ReplicateTest.php (hunk 1 above)
Comment (jordisala1991) [reply tests/Filesystem/ReplicateTest.php:22]: This file is not related to the current deprecation, this change is only about removing some master and slave words

File: src/DependencyInjection/SonataMediaExtension.php [hunk 2]
```diff
@@ -373,8 +373,9 @@ public function configureCdnAdapter(ContainerBuilder $container, array $config)
         }
 
         if ($container->hasDefinition('sonata.media.cdn.fallback') && isset($config['cdn']['fallback'])) {
+            // NEXT_MAJOR: Do not fallback to master
             $container->getDefinition('sonata.media.cdn.fallback')
-                ->replaceArgument(0, new Reference($config['cdn']['fallback']['master']))
+                ->replaceArgument(0, new Reference($config['cdn']['fallback']['primary'] ?? $config['cdn']['fallback']['master']))
```
Comment (ChristianGripp) [inline src/DependencyInjection/SonataMediaExtension.php:378]: You did not check this code

Comment (jordisala1991) [reply src/DependencyInjection/SonataMediaExtension.php:378]: Checked.

**Review by core23 — APPROVED** [review]

**Review by VincentLanglet — APPROVED** [review]

**Comment by VincentLanglet** [comment]
Thanks !

**Pull request merged**